diff --git a/PowerShell.Common.props b/PowerShell.Common.props
index dfc16f8..5c7d4c6 100644
--- a/PowerShell.Common.props
+++ b/PowerShell.Common.props
@@ -22,6 +22,10 @@
       <Output TaskParameter="ConsoleOutput" PropertyName="PowerShellVersion" />
     </Exec>
 
+    <PropertyGroup>
+        <NuGetAudit>false</NuGetAudit>
+    </PropertyGroup>
+
     <PropertyGroup Condition = "'$(ReleaseTag)' != ''">
       <!--
          DO NOT UPDATE without reviewing with the Microsoft update plan.
